journal-week-1.md

```diff
@@ -1,31 +1,10 @@
 ###Journal Week 1
 
-JavaScript consists of 4 primitive elements:
-*`null`
-* `Booleans`
-* `Numbers`
-* `Strings`
+Week one at the Iron Yard was a rush of new information. While somewhat familiar with JavaScript I found learning the different components and syntax helpful. However, having to figure out the logic and write my own functions was challenging. 
```

Comment:
Good essay! Would love to see you tackle bigger concepts like pair programming as well.